activities/views.py

```diff
@@ -14,12 +14,16 @@
 from rest_framework.response import Response
 from rest_framework import status
 
-#authentication
+# authentication
 from rest_framework.authentication import SessionAuthentication, BasicAuthentication, TokenAuthentication
 from rest_framework.permissions import IsAuthenticated
 
+# csrf
+from django.views.decorators.csrf import csrf_exempt
```

Comment:
unused import

Reply:
Code outdated. It has already been removed